src/Microsoft.AspNetCore.Hosting/GenericHost/GenericWebHost.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.AspNetCore.Hosting.Internal
```

Comment:
Eww `.Internal` - this is just noise

Reply:
Also folder and namespace haven't even met each other.

Reply:
Yea, I wanted to group the implementation together without sticking .GenericHost in the namespace name.

Reply:
put them in one file and use `#region`